src/nav/tutorial-infrastructure-monitoring.yml

```diff
@@ -0,0 +1,9 @@
+title: Diagnose resource bottlenecks
```

Comment:
`Diagnose resource bottlenecks` doesn't make me think "Infrastructure" intuitively. (I had trouble finding these docs in the deploy build).

`Diagnose infrastructure bottlenecks` may be a quick solution, but would love to hear other opinions from language writing.